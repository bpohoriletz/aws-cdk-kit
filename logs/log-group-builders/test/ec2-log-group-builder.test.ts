import Ec2LogGroupBuilder from '../ec2-log-group-builder';

describe('new Ec2LogGroupBuilder()', () => {
  it('has correct defaults', () => {
    expect(new Ec2LogGroupBuilder()).toMatchSnapshot();
  });
});
