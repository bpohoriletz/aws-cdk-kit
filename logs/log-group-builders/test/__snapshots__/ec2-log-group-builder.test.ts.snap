// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new Ec2LogGroupBuilder() has correct defaults 1`] = `
Ec2LogGroupBuilder {
  "props": LogGroupProduct {},
}
`;
